XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,84 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+		//Navigate to a URL and perform a long press on the non-highlighted address bar
+		navigator.goto(URLBarOpen)
+		app.textFields["address"].typeText("www.google.com\n")
+        waitUntilPageLoad()
+		app.textFields["url"].tap()
+		app.textFields["address"].press(forDuration: 2)
+
+		//Ensure that long press on address bar brings up a menu with Select All, Select, Paste, and Paste & Go
+        waitForExistence(app.menuItems["Select All"], timeout: 3)
+		XCTAssertTrue(app.menuItems["Select All"].exists)
+        XCTAssertTrue(app.menuItems["Select"].exists)
+        if UIPasteboard.general.string != nil {
+                XCTAssertTrue(app.menuItems["Paste"].exists)
+                XCTAssertTrue(app.menuItems["Paste & Go"].exists)
+            }
+
+        //Tap on Select option and make sure Copy, Cut, Paste, and Look Up are shown
+        app.menuItems["Select"].tap()
+        waitForExistence(app.menuItems["Copy"])
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
There is an existing function iPad() which checks this. So this can be changed as:
`if iPad() {`